### PR TITLE
CRW-1387 use /latest instead of /7.y.z in che.properties

### PR DESCRIPTION
**Pull request by nickboldt**
### What does this PR do?

* use /latest instead of 7.y.z in che.properties
* use sed_in_place to work on linux and macos
* make sed replacements more legible (less slash-escaping)
* update RELEASE.md; rename set_tag_version_images_linux.sh -> set_tag_version_images.sh

Change-Id: I6f0b586e4dc561d57d7c84b24d90031678fd3558
Signed-off-by: nickboldt <nboldt@redhat.com>

### Screenshot/screencast of this PR
N/A

### What issues does this PR fix or reference?

https://issues.redhat.com/browse/CRW-1387?_sscc=t

### How to test this PR?
N/A

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)

- [x] [The Eclipse Contributor Agreement is valid](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-eclipse-contributor-agreement-is-valid)
- [x] [Code produced is complete](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-produced-is-complete)
- [x] [Code builds without errors](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-builds-without-errors)
- [ ] [Tests are covering the bugfix](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#tests-are-covering-the-bugfix)
- [x] [The repository devfile is up to date and works, if one exists](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-repository-devfile-is-up-to-date-and-works)
- [x] [Sections What issues does this PR fix or reference and How to test this PR completed](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#sections-what-issues-does-this-pr-fix-or-reference-and-how-to-test-this-pr-completed)
- [x] [Relevant user documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [x] [Relevant contributing documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [x] [CI/CD changes implemented, documented and communicated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#cicd-changes-implemented-documented-and-communicated)

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### CODE REVIEW

**Review by ericwill — APPROVED** [review]
Looks good from the plugins side of things

**Comment by nickboldt** [comment]
[crw-ci-test]

**Comment by nickboldt** [comment]
Implied +1 from Sergii:

image

**Comment by nickboldt** [comment]
Followed up by https://github.com/eclipse/che-release/pull/22 in che-release overall script.

File: .ci/set_tag_version_images.sh [hunk 1]
```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+# Copyright (c) 2017-2021 Red Hat, Inc.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
```
Comment (matin2199) [inline .ci/set_tag_version_images.sh:6]: paul

Comment (nickboldt) [reply .ci/set_tag_version_images.sh:6]: are you saying "this should have EPL v2 on it?"

**Pull request merged**